config/client.lua

```diff
@@ -68,6 +68,6 @@ return {
 
     --- Only used by QB bridge
     hasKeys = function()
-        return exports['qb-vehiclekeys']:HasKeys()
+        return exports['qbx_vehiclekeys']:HasKeys()
```

Comment:
```suggestion
        return exports.qbx_vehiclekeys:HasKeys()
```
Since there is no dash you can call it directly without the []